### ISSUE TITLE
"The goal you specified requires a project to execute but there is no POM in this directory"

### ISSUE THREAD

**Issue by KPull** [nulldriver/maven-resource#43]
I've having trouble executing this resource with the following pipeline:

```yaml
jobs:
  - name: build
    plan:
      - get: source-code
        trigger: true

      - task: maven-package
        config:
          platform: linux
          image_resource:
            type: registry-image
            source:
              repository: maven
              tag: 3.6.3-openjdk-11
          inputs:
            - name: source-code
              path: source-code
          outputs:
            - name: binaries
              path: source-code/target
          run:
            dir: source-code
            path: mvn
            args:
              - package

      - put: package
        params:
          file: binaries/application-*.jar
          pom_file: source-code/pom.xml

    on_failure:
      put: notification
      params:
        message: ":warning: `application` has failed tests"
```

However, when running the Pipeline in Concourse, I get the following error:

```
The goal you specified requires a project to execute but there is no POM in this directory (/tmp/build/put). Please verify you invoked Maven from the correct directory. -> [Help 1]`
```

I've tried version 1.3.6 and the latest versions of this resource however both are giving us the same error. Can you please point out what I'm doing wrong, please?

**Comment by KPull** [comment]
Sorry, for posting an empty issue. I've now updated it with the relevant details.

**Comment by EndzeitBegins** [comment]
Hello @KPull,

sadly this resource seems to no longer being maintained actively by the author.
This issue is most likely due to more than one .jar-fie being picked up by your wildcard query for the `file` parameter: `binaries/application-*.jar`
The same happened to me when building a project with javadoc or sources.

This issue is known for some time, see #28.
Also there is a pull request to improve the error message but it hasn't been merged, see #39.

I hope this helps you somewhat.

**Comment by KPull** [comment]
Thanks for your help @EndzeitBegins.

This is a pity, given that this resource is "officially" one of the recommended resources on the Concourse homepage. It doesn't bode well for Concourse as a project if the recommended resources are not maintained.
